package.json

```diff
@@ -249,7 +249,7 @@
     "brfs": "^2.0.2",
     "browserify": "^16.5.1",
     "chalk": "^3.0.0",
```

Comment:
@Gudahtt Any concern with this version change for chromedriver?

Reply:
The last time I tried, CI consistently failed. That's the only reason it has stayed at v79.